tests/e2e/specs/in-game/game-over/rankedMatches.spec.js

```diff
@@ -5,15 +5,17 @@ import { playerOne, playerTwo, playerThree } from '../../../fixtures/userFixture
 import { Card } from '../../../fixtures/cards';
 
 const dayjs = require('dayjs');
+const utc = require('dayjs/plugin/utc');
+dayjs.extend(utc);
 
 function validateMatchResult(match, length, p1, p2, winnerId) {
   expect(match.player1.id).to.eq(p1);
   expect(match.player2.id).to.eq(p2);
-  expect(match.startTime).to.be.greaterThan(0);
+  expect(match.startTime).to.not.eq(null);
```

Comment:
Have we confirmed that the match will have a `null` `startTime` if none is set?

Reply:
Not yet, this could probably use some looking at

Reply:
Confirmed this will be null when unset.